lib/wallet/api/http/Cardano/Wallet/Api/Http/Server/Error.hs

```diff
@@ -184,6 +183,10 @@ liftHandler action = Handler (withExceptT toServerError action)
 liftE :: IsServerError e => e -> Handler a
 liftE = liftHandler . throwE
 
+-- | Lift an IO action into servant 'Handler'
+handler :: IO a -> Handler a
+handler = Handler . lift
```

Comment:
```suggestion
handler :: IO a -> Handler a
handler = liftIO
```

In the module `Cardano.Wallet.Api.Http.Shelley.Server`, we could use `liftIO` directly instead of `handler`, but I think that using the synonym `handler` is perfectly fine for the time being.